src/routers/user/index.ts

```diff
@@ -75,7 +75,7 @@ router.post('/thirdPartyAuth', async (req: Request, res: Response) => {
 // logout
 router.post(
   '/logout',
-  auth,
+  auth(),
```

Comment:
Just to clarify, does invoking this with no param mean that I am not admin? how would the admin routes be triggered?

Reply:
Invoking with not params is exactly as it was before so no admin checking.
Invoking with `{ isAdmin: true }` requires `isAdmin` set to true on the user doc to use route.